src/sentry/api/endpoints/organization_details.py

```diff
@@ -382,8 +389,8 @@ def save(self):
             org.flags.require_email_verification = self.initial_data["requireEmailVerification"]
         if "name" in self.initial_data:
```

Comment:
out of curiosity, why do we not use the `validated_data` instead of `initial_data` for everything else here?

Reply:
@maxiuyuan I was curious about that too. I think we rely on the serializers to validate the data. And when the data is validated, then `initial_data` is also "validated".

Reply:
Ideally we would use `validated_data`, but if the serializer doesn't mutate the data at all as part of the validation process then they're equivalent.